src/tsconfig.json

```diff
@@ -17,11 +16,9 @@
         ],
         "allowJs": true,
         "downlevelIteration": true,
-        "noEmit": true,
+        "noImplicitAny": true,
+        "resolveJsonModule": true,
         "jsx": "react",
-        "jsxFactory": "m"
-    },
-    "exclude": [
```

Comment:
This should be back in?

Reply:
It wasn't making a difference in the compile. I can add it back in though.